conda/meta.yaml

```diff
@@ -1,22 +1,27 @@
+{% set name = "zstash" %}
+{% set version = "0.4.0" %}
```

Comment:
These Jinja2 variables let you reuse the same name and version in multiple places.  Standard on conda-forge.